dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05180UpdateFriendlyNameField.java

```diff
@@ -0,0 +1,37 @@
+package com.dotmarketing.startup.runonce;
+
+import static com.dotmarketing.portlets.htmlpageasset.business.HTMLPageAssetAPI.FRIENDLY_NAME_FIELD;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import com.dotmarketing.util.Logger;
+
+/**
+ * @author nollymar
+ */
+public class Task05180UpdateFriendlyNameField implements StartupTask {
+
+    private final String UPDATE_FIELD = "UPDATE field set velocity_var_name='" + FRIENDLY_NAME_FIELD
```

Comment:
I think this should be more specific, joining to the structure table and adding a `where structuretype=5`

Reply:
meaning, only update the `friendlyname` fields on htmlpages

Reply:
good catch!

Reply:
done